src/kusto/azext_kusto/manual/_params.py

```diff
@@ -0,0 +1,573 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.

```

Comment:
I'm curious. Is this file written manually?

Reply:
No, anything under manual folder except __init__.py is manual added.

Reply:
I edit the generated _params file and added it to the manual folder